examples/users-example.js

```diff
@@ -0,0 +1,154 @@
+require('dotenv').config({
+	path: "../.env"
+});
+
+var SeniorApi = require('../built/index').default;
+
+var username = process.env.SENIOR_USERNAME;
+var password = process.env.PASS;
+
+var api = new SeniorApi(username, password);
+
+api.authentication.login().then(function (json) {
+	var jsonToken = JSON.parse(json.body.jsonToken);
+	api.accessToken = jsonToken.access_token;
+
+	var username = "teste-user";
```

Comment:
Substituir valores para opcionais ao DEV. Exemplo:
```suggestion
	var username = "<NOME_USUARIO_PARA_TESTES>";
```